ModernFlyouts/MultilingualResources/ModernFlyouts.ru.xlf

```diff
@@ -418,32 +418,29 @@
         </trans-unit>
         <trans-unit id="Settings.AppTheme" translate="yes" xml:space="preserve">
           <source>App Theme</source>
-
-          <target state="new">Тема приложения</target>
+          <target state="needs-review-translation" state-qualifier="mt-suggestion">Тема приложения</target>
         </trans-unit>
         <trans-unit id="Settings.UseColoredTrayIcon" translate="yes" xml:space="preserve">
           <source>Use colored tray icon</source>
-          <target state="new">Использовать цветной значок в трее</target>
-          
+          <target state="needs-review-translation" state-qualifier="mt-suggestion">Используйте цветной значок лотка</target>
         </trans-unit>
         <trans-unit id="NgsSettings_Reset" translate="yes" xml:space="preserve">
           <source>ngs.Settings_Reset}</source>
           <target state="needs-review-translation" state-qualifier="mt-suggestion">Ngs. Settings_Reset»</target>
         </trans-unit>
         <trans-unit id="Settings.Reset" translate="yes" xml:space="preserve">
           <source>Reset</source>
-          
-          <target state="new">Сброс</target>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">Сбросить</target>
         </trans-unit>
         <trans-unit id="Settings.ResetToDefaults" translate="yes" xml:space="preserve">
           <source>Reset all settings to defaults</source>
-          <target state="new">Сбросить все настройки до значений по умолчанию</target>
-          
```

Comment:
@Samuel12321 I think we have our faulty line.
It was missing a closing `</trans-unit>` tag

It was also due to the merge commit you did before merging in #194 
Seems like GitHub conflict manager is dumb 🤷